src/containers/PaletteContainer.js

```diff
@@ -23,26 +27,34 @@ export default class PaletteContainer extends Container {
     window.addEventListener('resize', this.onResize);
   }
 
+  setStateAndStorage = (data, cb = () => {}) => {
+    this.setState(data, () => {
+      cb();
```

Comment:
Since this callback is fired synchronously, does it make sense for it to be invoked _after_ we set the value in local storage?

My concern is:

```js
setStateAndStorage({ foo: 'bar', ()=> {
  const val = localStorage.getItem(STATE_KEY) // <-- this will be "old" data
  doStuffWithValue(val)
})
```